boss-ws/src/main/java/org/genomebridge/boss/http/resources/FCSResource.java

```diff
@@ -2,25 +2,31 @@
 
 import java.util.concurrent.ConcurrentHashMap;
 
-import javax.ws.rs.DELETE;
-import javax.ws.rs.GET;
-import javax.ws.rs.HEAD;
-import javax.ws.rs.HeaderParam;
-import javax.ws.rs.PUT;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
+import javax.ws.rs.*;
 import javax.ws.rs.core.HttpHeaders;
 import javax.ws.rs.core.Response;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.wordnik.swagger.annotations.*;
 
 @Path("fcs/{objName : [^?]+}")
 @JsonInclude(Include.NON_NULL)
+@Api(value = "fcs", description = "Operations on object's data",produces = "application/json")
 public class FCSResource {
 
     @GET
-    public Response getData(@PathParam("objName") String objName) {
+    @Produces("application/json")
+    @ApiOperation(value = "Querying for Object's data by objName",
+                  httpMethod = "GET")
```

Comment:
The response value here should be able to be modeled as (warning ... untested):
response = Byte.class
responseContainer = "Array"